src/AdapterPluginManager.php

```diff
@@ -45,7 +58,7 @@ class AdapterPluginManager extends AbstractPluginManager
      * @return void
      * @throws Exception\RuntimeException if invalid
      */
-    public function validatePlugin($plugin)
+    public function validate($plugin)
```

Comment:
Ok, but in this way we are going to change the exception from `Zend\Serializer\Exception\RuntimeException` to `Zend\ServiceManager\Exception\InvalidServiceException`.